python/src/trezorlib/cli/__init__.py

```diff
@@ -68,8 +73,11 @@ def get_transport(self) -> Transport:
         # if this fails, we want the exception to bubble up to the caller
         return get_transport(self.path, prefix_search=True)
 
-    def get_ui(self) -> ClickUI:
-        return ClickUI(passphrase_on_host=self.passphrase_on_host)
+    def get_ui(self) -> Union[ClickUI, ScriptUI]:
```

Comment:
this should presumably be returning `TrezorClientUI`

Reply:
Done in e5670a4